website/routes.py

```diff
@@ -330,6 +330,16 @@ def make_url_map(app):
             'get', website_views.get_dashboard, json_renderer),
     ], prefix='/api/v1')
 
+    process_rules(app, [
+        Rule('/dashboard/confirmed_emails/', 'get', auth_views.confirm_user_get, json_renderer),
+        Rule('/dashboard/confirmed_emails/', 'put', auth_views.add_confirmed_email, json_renderer)
+
+    ], prefix='/api/v1')
+
+    process_rules(app, [
+        Rule('/dashboard/remove_confirmed_emails/', 'put', auth_views.confirm_email_remove, json_renderer)
```

Comment:
Because this is `remove`-type functionality, it might be better as a `DELETE` request on the same endpoint as `confirm_user_get` and `add_confirmed_email`: 
```python
process_rules(app, [
    Rule('/dashboard/confirmed_emails/', 'get', auth_views.confirm_user_get, json_renderer),
    Rule('/dashboard/confirmed_emails/', 'put', auth_views.add_confirmed_email, json_renderer),
    Rule('/dashboard/confirmed_emails/', 'delete', auth_views.confirm_email_remove, json_renderer)
], prefix='/api/v1')
```